airflow/cli/commands/kerberos_command.py

```diff
@@ -33,6 +34,17 @@ def kerberos(args):
     """Start a kerberos ticket renewer."""
     print(settings.HEADER)
 
+    mode_mapping = {
+        "daemon": KerberosMode.DAEMON,
+        "one-time": KerberosMode.ONE_TIME,
+    }
+    if args.mode:
```

Comment:
I believe we also have to describe the mode similar to other CLI switches so that it shows up in the --help output?

Reply:
Apologies could not find that. Can you point me to it?

Reply:
Nevermind. I found it and fixed it